docs/content/en/docs/configuration/keys.md

```diff
@@ -1999,6 +2000,7 @@ The following keys are supported:
 * `modsecurity-timeout-idle`: Defines the maximum time to wait before close an idle connection. Default value is `30s`.
 * `modsecurity-timeout-processing`: Defines the maximum time to wait for the whole ModSecurity processing. Default value is `1s`.
 * `modsecurity-timeout-server`: Defines the maximum time to wait for an agent response. Configures the haproxy's timeout server. Defaults to `5s` if not configured.
+* `modsecurity-args`: Space separated list of arguments that HAProxy will send to the modsecurity agent. You can override this to e.g. prevent sending the request body to modsecurity which will improve performance, but reduce security. The arguments must be valid HAProxy "[fetches](https://www.haproxy.com/documentation/hapee/latest/configuration/fetches/overview/)".
```

Comment:
Use alphabetical order here as well. Use the HAProxy community doc instead - although I'm aware that it's more a ref guide than a user doc. [This one](https://docs.haproxy.org/2.4/configuration.html#7.3) should fit.

You can also remove the double quotes and change "fetches" to "sample fetch methods" (without the quotes).

Reply:
Missing updating the link and link text here.